### ISSUE TITLE
Improved SVG handling

### ISSUE THREAD

**Issue by chaoaretasty** [RealFaviconGenerator/realfavicongenerator#65]
I noticed a couple of annoyances while using SVG images that I think can easily be worked around.

Firstly the SVG's declared size is used in the sizing algorithm to work out what images it can generate. As SVG is a vector format I think it's reasonable to ignore this and generate all possible sizes considering the relatively arbitrary nature of an SVG's pixel size.

Secondly I think it could be possible to turn off the requirement for an exactly square image, especially as an SVG may well be sized to the edge of the image and contain transparency. In such cases it could be centred and have the rest padded with transparency. Perhaps highlight when uploading an non-square image that this has happened.

Neither are particularly troublesome to work around but for this format in particular these seem reasonable defaults.

**Comment by phbernard** [comment]
Thank you for the feedback.

Regarding the size: you are totally right. I've just tried with a 128x128 SVG picture and the experience is painful. Not only there is this irrelevant warning message, but high resolution icons (eg. 152x152 Apple touch icon) are not generated. Good old bug. It was fixed and deployed a minute ago.

Regarding the non-square pictures: centered... padded... hum, this sounds tough :) I can't address this at the moment. I have created issue #71 to separated the two issues you submitted.
